lib/ASTGen/Sources/ASTGen/ASTGen.swift

```diff
@@ -373,7 +261,7 @@ extension ASTGenVisitor {
       return nil
     }
 
-    return self.generate(node)
+    return self.generate(type: node)
```

Comment:
If we're going to give these argument labels, should the optional variants also get them? Or is the intention to avoid the risk of recursion?

Reply:
Yes, I will add argument labels to all the existing `generate(_:)` functions in follow-ups.